docs/developer-docs/latest/getting-started/usage-information.md

```diff
@@ -40,21 +40,28 @@ The collected data are of a non-sensitive nature and no personal data is collect
 
 ### Opt-out
 
-You can easily disable the default data collection feature.
-
-Should you decide to opt-out, you may do so by removing the `uuid` property in the `package.json` file located within the root of your project. This will automatically disable this feature.
-
-```json
-{
-  "name": "my-project",
-  "version": "0.1.0",
-  "strapi": {
-    "uuid": "7b581c0d-89b7-479e-b379-a76ab90b8754"
-  },
-  "license": "SEE LICENSE IN LICENSE"
-}
+The default data collection feature can be disabled using the following CLI command. Alternatively, the `telemetryDisabled: true` flag in the project `package.json` file will also disable data collection. Deleting the flag or setting it to false will re-enable data collection.
```

Comment:
You could move the second sentence ("Alternatively [...]") to after the code block. It could be a bit more fluid to have the code block with CLI command right after "using the following CLI command".